### ISSUE TITLE
JAWS does not announce the state information properly for "Expand/collapse" and "More" button.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1026]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1216326

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 
Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" Menu item and select it.
Navigate through all the controls under "Data Explorer" blade.
Turn on JAWS.
Navigate to "Expand/Collapse" buttons and 'More' menu(...) button present after collapse tree button using tab key and activate them.
Observe the screen reader behavior
Verify the issue.

Actual Result:
JAWS does not announce the state information properly for "Expand/collapse" and current state as 'collapsed' for "More" button.

Expected Result:
JAWS should announce the state information properly for "Expand/collapse" and "More" button.

Note:
This issue does not exist with Narrator and NVDA.

https://user-images.githubusercontent.com/8519407/132720980-0ba5c0c1-31f1-46ae-bf26-ca8001363fe8.mp4
